its/ruling/src/test/resources/autoscan/autoscan-diff-by-rules.json

```diff
@@ -361,9 +361,9 @@
   },
   {
     "ruleKey": "S1181",
-    "truePositives": 3,
-    "falseNegatives": 12,
-    "falsePositives": 0
+    "truePositives": 0,
+    "falseNegatives": 10,
+    "falsePositives": 3
```

Comment:
I don't understand why S1181 enters into the picture and why we have 3 new false positives.